src/aleph/sdk/client.py

```diff
@@ -608,22 +613,85 @@ async def get_posts(
             resp.raise_for_status()
             return await resp.json()
 
+    async def download_file_to_buffer(
+            self,
+            file_hash: str,
+            output_buffer: BinaryIO,
+            chunk_size: int,
+    ) -> None:
+        """
+        Download a file from the storage engine and write it to the specified output buffer.
+        :param self: The current instance of the class.
+        :param file_hash: The hash of the file to retrieve.
+        :param output_buffer: The binary output buffer to write the file data to.
+        :param chunk_size: Size of the chunk to download.
+        """
+        async with aiohttp.ClientSession() as session:
+            async with session.get(f"https://api1.aleph.im/api/v0/storage/raw/{file_hash}") as response:
```

Comment:
Using our load balancer would help keeping this work when server api1 is under maintenance.
```suggestion
            async with session.get(f"https://official.aleph.im/api/v0/storage/raw/{file_hash}") as response:
```

There is already a setting in the SDK that points to the API server, `settings.API_HOST`.

Reply:
Use `self.http_session`, we initialize one session when creating the `AlephClient` object.